Library/Homebrew/dev-cmd/create.rb

```diff
@@ -141,20 +141,19 @@ def create_cask(args:)
 
   def create_formula(args:)
     fc = FormulaCreator.new(args)
+    fc.url = args.named.first
     fc.name = if args.set_name.blank?
-      stem = Pathname.new(args.named.first).stem.rpartition("=").last
-      print "Formula name [#{stem}]: "
-      __gets || stem
+      name = url2name(fc.url)
```

Comment:
```suggestion
      name = fc.name_from_url
```
or something seems appropriate

Reply:
Renamed.